src/modules/explain.spec.js

```diff
@@ -0,0 +1,133 @@
+import reducer, {
+  switchToTreeView,
+  switchToJSONView,
+  explainStateChanged,
+  explainPlanFetched,
+  SWITCHED_TO_TREE_VIEW,
+  SWITCHED_TO_JSON_VIEW,
+  EXPLAIN_STATE_CHANGED,
+  EXPLAIN_PLAN_FETCHED
+
+} from 'modules/explain';
+
+const explainSpy = {
```

Comment:
Not to be nitpicky on nomenclature don't think this is a "spy". :) Outside of that LGTM.

Reply:
good point! updated